openIt/ngsild-payloads/key.jsonld

```diff
@@ -21,8 +21,14 @@
     "type": "Property",
     "value": "P1D"
   },
-  "category": {
-    "type": "Property",
-    "value": "Fire Department"
-  }
+  "isAvailableForOrganization": [
+    {
+      "type": "Property",
+      "value": "Agglomération Provence Verte"
```

Comment:
I would instead see this as a relationship

Reply:
Agreed. Will update with new Organization type